crates/ruff_python_parser/src/lib.rs

```diff
@@ -234,7 +234,7 @@ pub fn parse_unchecked_source(source: &str, source_type: PySourceType) -> Parsed
 }
 
 /// Represents the parsed source code.
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Clone, Default)]
```

Comment:
Can you explain me the use case for adding a `Default` implementation for `Parsed` and `Tokens` ?